README.md

```diff
@@ -12,22 +12,44 @@
 Litmus is a command line tool that allows you to run acceptance tests against Puppet modules.
 
 Litmus allows you to:
-* Provision targets to test against
-* Install a Puppet agent
-* Install a module
-* Run tests
-* Tear down the infrastructure
+
+- Provision targets to test against
+
+- Install a Puppet agent
+
+- Install a module
+
+- Run tests
+
+- Tear down the infrastructure
```

Comment:
are these new newlines intentional?

Reply:
No. I didn't noticed. These lines were added added by vscode when I hit save. I just look on Readme and seems to look exactly as Readme from the main branch.